enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/dependency/DependencyConvergence.java

```diff
@@ -50,6 +53,8 @@ public final class DependencyConvergence extends AbstractStandardEnforcerRule {
 
     private List<String> excludes;
 
+    private List<String> scopes = Arrays.asList(SCOPE_COMPILE, SCOPE_RUNTIME, SCOPE_IMPORT, SCOPE_SYSTEM);
```

Comment:
SCOPE_IMPORT - is special one used in dependencyManagement - we don't dependencies in this scope.
Please remove it.

Reply:
Removed.